[2025-01-15T13:43:48.536+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: ETL_influencer_marchendise.data_load manual__2025-01-15T13:43:28.890192+00:00 [queued]>
[2025-01-15T13:43:48.541+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: ETL_influencer_marchendise.data_load manual__2025-01-15T13:43:28.890192+00:00 [queued]>
[2025-01-15T13:43:48.541+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2025-01-15T13:43:48.541+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 2
[2025-01-15T13:43:48.541+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2025-01-15T13:43:48.548+0000] {taskinstance.py:1389} INFO - Executing <Task(BashOperator): data_load> on 2025-01-15 13:43:28.890192+00:00
[2025-01-15T13:43:48.551+0000] {standard_task_runner.py:52} INFO - Started process 632 to run task
[2025-01-15T13:43:48.553+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'ETL_influencer_marchendise', 'data_load', 'manual__2025-01-15T13:43:28.890192+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/P2M3_afif_makruf_DAG.py', '--cfg-path', '/tmp/tmpu6v9q4kb', '--error-file', '/tmp/tmpq9i8uhi4']
[2025-01-15T13:43:48.554+0000] {standard_task_runner.py:80} INFO - Job 4: Subtask data_load
[2025-01-15T13:43:48.590+0000] {task_command.py:371} INFO - Running <TaskInstance: ETL_influencer_marchendise.data_load manual__2025-01-15T13:43:28.890192+00:00 [running]> on host ced01879ea9b
[2025-01-15T13:43:48.627+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=afif
AIRFLOW_CTX_DAG_ID=ETL_influencer_marchendise
AIRFLOW_CTX_TASK_ID=data_load
AIRFLOW_CTX_EXECUTION_DATE=2025-01-15T13:43:28.890192+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2025-01-15T13:43:28.890192+00:00
[2025-01-15T13:43:48.627+0000] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2025-01-15T13:43:48.628+0000] {subprocess.py:74} INFO - Running command: ['/bin/bash', '-c', 'sudo -u *** python /opt/***/dags/load.py']
[2025-01-15T13:43:48.633+0000] {subprocess.py:85} INFO - Output:
[2025-01-15T13:43:57.124+0000] {subprocess.py:96} INFO - Command exited with return code 0
[2025-01-15T13:43:57.155+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=ETL_influencer_marchendise, task_id=data_load, execution_date=20250115T134328, start_date=20250115T134348, end_date=20250115T134357
[2025-01-15T13:43:57.209+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2025-01-15T13:43:57.231+0000] {local_task_job.py:279} INFO - 0 downstream tasks scheduled from follow-on schedule check
